pdr_backend/lake/duckdb_data_store.py

```diff
@@ -243,34 +247,138 @@ def query_data(self, query: str) -> Optional[pl.DataFrame]:
+        """
+        query = self.get_query_drop_records_from_table_by_id(
+            drop_table_name, ref_table_name
+        )
+        self.execute_sql(query)
+
+    @enforce_types
+    def get_query_move_table_data(self, from_table, to_table):
+        """
+        Builds the query string to perform the operation and returns it
+        @arguments:
+            from_table - where we'll be taking data from
+            to_table - wehere we'll be inserting data
         @example:
             move_table_data("temp_people", "people")
         """
 
         # Check if the table exists
         table_names = self.get_table_names()
+        assert (
+            from_table.table_name in table_names
+        ), f"Table {from_table.table_name} is "
 
-        if temp_table.fullname not in table_names:
-            logger.info(
-                "move_table_data - Table %s does not exist", temp_table.fullname
-            )
-
-        permanent_table_name = permanent_table.fullname
         # check if the permanent table exists
-        if permanent_table_name not in table_names:
-            # create table if it does not exist
-            self.execute_sql(
-                f"CREATE TABLE {permanent_table_name} AS SELECT * FROM {temp_table.fullname}"
-            )
-            return
+        if to_table.table_name not in table_names:
+            return f"CREATE TABLE {to_table.table_name} AS SELECT * FROM {from_table.table_name};"
 
         # Move the data from the temporary table to the permanent table
+        return (
+            f"INSERT INTO {to_table.table_name} SELECT * FROM {from_table.table_name};"
+        )
+
+    @enforce_types
+    def move_table_data(self, from_table, to_table):
+        """
+        Move the table data from the temporary storage to the permanent storage.
+        @arguments:
+            from_table - where we'll be taking data from
+            to_table - wehere we'll be inserting data
+        @example:
+            move_table_data("temp_people", "people")
+        """
+
+        # Check if the table exists
+        move_table_query = self.get_query_move_table_data(from_table, to_table)
+        self.execute_sql(move_table_query)
+
+    @enforce_types
+    def fill_table_from_csv(self, table_name: str, csv_folder_path: str):
+        """
+        Insert to table from CSV files.
+        @arguments:
+            table_name - A unique name for the table.
+            csv_folder_path - The path to the folder containing the CSV files.
+        @example:
+            fill_table_from_csv("data/csv", "people")
+        """
+
+        csv_files = glob.glob(os.path.join(csv_folder_path, "*.csv"))
+
+        logger.info("csv_files %s", csv_files)
+        for csv_file in csv_files:
+            df = pl.read_csv(csv_file)
+            self.insert_from_df(df, table_name)
+
+    @enforce_types
+    def update_data(self, df: pl.DataFrame, table_name: str, column_name: str):
```

Comment:
not sure if this is an addition or simply a conflict with main. I remember removing some of these functions: fill_table_from_csv, update_data etc. Since they were either unused or needed renaming/simplification. Did you add these on purpose? Would it be of help to you for me to search for the PR where I did the simplifications and help out with that reference?

Reply:
yes, I did add this as part of new requirements for the ETL
I have also moved the CRUD functions to the CRUD object

it will be easier to review this inside of the new PR [#1423](https://github.com/oceanprotocol/pdr-backend/pull/1423)